### PR TITLE
Update twin.macro installation instructions

### PR DESCRIPTION
**Pull request by ben-rogerson**
## Description

Twin.macro is now out of alpha and so I've updated the usage example.

### CODE REVIEW

File: docs/docs/tailwind-css.md [hunk 1]
```diff
@@ -94,34 +92,24 @@ module.exports = {
...
-
-// Install tailwind 0.7.4 and stable tailwind.macro
-npm install tailwindcss@0.7.4
-npm install tailwind.macro
-```
-
-2. Use the Babel Macro (`tailwind.macro`) in your styled component
+4. Use twin.macro to create your styled component
 
 ```javascript
```
Comment (marcysutton) [inline docs/docs/tailwind-css.md:97]: Users could be a little confused about where this goes, so we typically advise showing a complete component. Might as well update it while we're in there.
````suggestion
```jsx:title=src/pages/index.js
````

File: docs/docs/tailwind-css.md [hunk 2]
```diff
@@ -94,34 +92,24 @@ module.exports = {
...
-import styled from "@emotion/styled"
-import tw from "tailwind.macro"
+import tw, { styled } from "twin.macro"
```
Comment (marcysutton) [inline docs/docs/tailwind-css.md:98]:
```suggestion
import React from "react"
import tw, { styled } from "twin.macro" //highlight-line
```

File: docs/docs/tailwind-css.md [hunk 3]
```diff
@@ -94,34 +92,24 @@ module.exports = {
...
+  ${tw`bg-blue-500 hover:bg-blue-800 text-white p-2 rounded`}
 `
 
-// tailwind.macro@next
+// or use the shorthand version
+
 const Button = tw.button`
-  bg-blue hover:bg-blue-dark text-white p-2 rounded
+  bg-blue-500 hover:bg-blue-800 text-white p-2 rounded
 `
```
Comment (marcysutton) [inline docs/docs/tailwind-css.md:108]:
```suggestion
`

const IndexPage = () => (
  <div>
    <h1>Hi people</h1>
    <Button>Activate</Button> // highlight-line
  </div>
)

export default IndexPage
```

**Comment by ben-rogerson** [comment]
Thanks for the great suggestions Marcy, over to you 👍

**Pull request merged**